CodeStarterCamp_Week1/main.swift

```diff
@@ -1,18 +1,59 @@
 import Foundation
 
-let iceCreamRow = String(repeating: "*", count: 11)
+orderAndDraw()
 
-func drawIceCream() {
-    for _ in 0..<8 {
-        print(iceCreamRow)
+//길이(Int)에 따른 막대(빼빼로 손잡이)를 그리는 함수
+func drawPeperoStick(_ stickHeight: Int, _ peperoRow: String) {
+    let peperoRowLen: Int = peperoRow.count
```

Comment:
변수이름이 길어져서 Len으로 줄여쓰셨군요
https://www.swift.org/documentation/api-design-guidelines/
약어를 사용하지 말라고 나와있네요🥲

Reply:
Avoid abbreviations. Abbreviations, especially non-standard ones, are effectively terms-of-art, because understanding depends on correctly translating them into their non-abbreviated forms.

제대로 읽지 않았네요 제가.. 😓